jld-day15-parser/Cargo.toml

```diff
@@ -2,5 +2,6 @@
 name = "day15"
 version = "0.1.0"
 authors = ["Jed Davis <jld@panix.com>"]
+
 [dependencies]
-combine = "1.2.0"
+combine = "= 1.3.0"
```

Comment:
This may be correct, but is the change from 1.2.0 to 1.3.0 intentional? I think Cargo may have already been using 1.3.0, so this is probably fine...